### PR TITLE
ED flowsheet updates

### PR DESCRIPTION
**Pull request by lbibl**
## Summary/Motivation:
To update a few changes in the ED flowsheet (with conc. channel recirculation feature) for better solving and debugging. 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: watertap/examples/flowsheets/electrodialysis/tests/test_electrodialysis_1stack_conc_recirc.py [hunk 1]
```diff
@@ -114,24 +115,22 @@ def test_optimization(self):
...
-        assert value(m.fs.mem_area) == pytest.approx(13.8159, rel=1e-3)
-        assert value(m.fs.EDstack.cell_pair_num) == pytest.approx(14, rel=1e-8)
-        assert value(m.fs.EDstack.cell_length) == pytest.approx(5.0094, rel=1e-3)
-        assert value(m.fs.EDstack.voltage_applied[0]) == pytest.approx(
-            16.8753, rel=1e-3
-        )
+        assert value(m.fs.mem_area) == pytest.approx(15.1283, rel=1e-3)
+        assert value(m.fs.EDstack.cell_pair_num) == pytest.approx(16, rel=1e-8)
+        assert value(m.fs.EDstack.cell_length) == pytest.approx(4.800, rel=1e-3)
+        assert value(m.fs.EDstack.voltage_applied[0]) == pytest.approx(18.785, rel=1e-3)
```
Comment (adam-a-a) [inline watertap/examples/flowsheets/electrodialysis/tests/test_electrodialysis_1stack_conc_recirc.py:121]: @lbibl do we know what triggered the change in results?

**Review by adam-a-a — APPROVED** [review]
LGTM - I had one question about the change in some results but it probably shouldn't hold up the PR.

**Comment by lbibl** [comment]
Yes. There was a bug in the original that was discovered during the analysis work.  The correction is the change of pump1 to pump0 in L266.  The bug in the original let pressure drop not calculated in one channel and correcting it resulted in a minor variation in other optimization outcomes.

**Review by hunterbarber — APPROVED** [review]
Pretty straightforward changes, LGTM

**Pull request merged**